test/browser/features/error-correlation.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Error Correlation
+
+  Scenario: Error does not include the correlation property when no spans are open
+    Given I navigate to the test URL "/error-correlation"
+    And I wait to receive a sampling request
+    And I click the element "send-error"
+    And I wait to receive an error
+    Then the error is a valid browser payload for the error reporting API
+    And the error payload field "events.0.correlation" is null
+
+  Scenario: Reported errors include the current trace and span id
+    Given I navigate to the test URL "/error-correlation"
+    And I wait to receive a sampling request
+    Then I click the element "start-span"
+    And I click the element "send-error"
+
+    # Wait for the error
+    And I wait to receive an error
+    Then the error is a valid browser payload for the error reporting API
+
+    # End the span
+    Then I click the element "end-span"
+    And I wait for 1 span
+
+    # Assert on the span, and store the trace id and span id
+    Then the trace payload field "resourceSpans.0.scopeSpans.0.spans.0.id" is stored as the value "spanId"
+    And the trace payload field "resourceSpans.0.scopeSpans.0.spans.0.traceid" is stored as the value "traceId"
+
+    # Check span id is included in the error
+    And the error payload field "events.0.correlation.spanId" equals the stored value "spanId"
+    And the error payload field "events.0.correlation.traceId" equals the stored value "traceId"
+
+  Scenario: Reported errors do not include the trace and span id if the span is not the current context
```

Comment:
@yousif-bugsnag  does this make a bit more sense? and am I missing something simple here? it's failing on like 62 at the moment

Reply:
yeah that definitely makes more sense 👍🏾 

not sure why that is failing though as it seems sensible enough 🤔 would need to take a look at the traces to debug it